x-pack/plugins/observability/public/components/shared/core_web_vitals/core_vital_item.tsx

```diff
@@ -81,19 +82,22 @@ export function CoreVitalItem({
   ranks = [100, 0, 0],
   isCls,
   helpLabel,
+  hasVitals = false,
 }: Props) {
   const palette = euiPaletteForStatus(3);
 
   const [inFocusInd, setInFocusInd] = useState<number | null>(null);
 
   const biggestValIndex = ranks.indexOf(Math.max(...ranks));
 
-  if ((value === null || value !== undefined) && ranks[0] === 100 && !loading) {
+  if (!value && !hasVitals && !loading) {
     return <EuiCard title={title} isDisabled={true} description={NO_DATA} />;
   }
+
   return (
     <>
       <EuiStat
+        aria-label={`${title} ${value}`}
```

Comment:
EUI Stat uses the title and description and concats them in order to make accessible text. It only does this, however, when both title and description are strings. Since description is a component in this case, it did not create accessible text for us. Previously, this entire stat, both the title and the value, were `aria-hidden`. Adding `aria-label` to the component ensures that accessible text is provided.

Reply:
Can you add this as a comment with the code so it can live alongside it? Also, is this a potential EUI bug? I'm not sure if there's a clean way for EUI to handle this case, but if there is, we should file one.

Reply:
I don't think it's a bug, because EUI specifically in their documentation exposes `aria-label` as a prop for EuiStat for this type of "escape hatch" to add in an accessible label.